packages/eslint-plugin-gestalt/src/prefer-list.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * @fileoverview Prefer List: Prevent ul or ol list tags for basic use cases.
+ */
+
+// @flow strict
+import { type ESLintRule } from './helpers/eslintFlowTypes.js';
+
+const rule: ESLintRule = {
+  meta: {
+    type: 'suggestion',
+    docs: {
+      description:
+        'Prefer List: Prevent ul or ol list tags for basic use cases. Use Gestalt List, instead',
+      category: 'Gestalt alternatives',
+      recommended: true,
+      url: 'https://gestalt.pinterest.systems/web/list',
```

Comment:
can we put here

`https://gestalt.pinterest.systems/eslint%20plugin#gestaltprefer-list` ?

Reply:
DONE